src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedFieldSymbol.cs

```diff
@@ -92,7 +92,7 @@ internal override bool IsNotSerialized
             }
         }
 
-        internal override bool HasPointerType => _underlyingField.HasPointerType;
+        internal sealed override bool HasPointerType => _underlyingField.HasPointerType;
```

Comment:
>sealed [](start = 17, length = 6)

This change doesn't feel appropriate. In fact this override doesn't feel appropriate in general because we do not override Type property and both should be consistent. I suggest reverting this change and either making a change in a different PR, or opening an issue to follow up.